src/sentry/web/frontend/accounts.py

```diff
@@ -273,12 +273,10 @@ def recover_confirm(request, user_id, hash, mode="recover"):
 
 # Set password variation of password recovery
 set_password_confirm = partial(recover_confirm, mode="set_password")
-set_password_confirm = update_wrapper(set_password_confirm, recover)
 
 
 # Relocation variation of password recovery
 relocate_confirm = partial(recover_confirm, mode="relocate")
-relocate_confirm = update_wrapper(relocate_confirm, recover)
```

Comment:
@evanpurkhiser it's unclear why `update_wrapper` was here but I traced it to e962e56a912e27f79d6acc3b77cf17fe1f34a313

it's also odd because `update_wrapper` is ~typically only used when writing a decorator -- though this is a `functools.partial` -- it's also odd because the "copied from" function is `recover` -- which is not the function that's being wrapped

either way, mypy was mad about this so I removed the `update_wrapper` because it doesn't seem to be doing anything?

Reply:
I wish I had a better answer, but I wrote this code almost 7 years ago 😬

Reply:
tests pass so yolo

Reply:
my only guess is python 2 multiprocessing something something but even that seems unlikely